deploy/microshift-bootc/config-svc/app.py

```
@@ -0,0 +1,2302 @@
+            return f'{days}d{hours}h' if hours > 0 else f'{days}d'
+    except Exception as e:
+        print(f"Error calculating age: {e}", file=sys.stderr)
+        return 'N/A'
+
+
+def get_default_route_ip():
+    """Get the IP address of the default route interface."""
+    try:
+        # Get default route
+        result = subprocess.run(
+            ['ip', 'route', 'show', 'default'],
```

Ruff (0.14.8)

Starting a process with a partial executable path

(S607)